src/tools/compiletest/src/util.rs

```diff
@@ -113,6 +113,8 @@ pub fn matches_env(triple: &str, name: &str) -> bool {
 pub fn get_pointer_width(triple: &str) -> &'static str {
```

Comment:
Huh I would've expected `compiletest` to get this from `rustc` (using `--print`).

Reply:
Yeah - writing this line made me feel very dirty.